csrc/scheduler/tools/inlining.h

```diff
@@ -113,4 +113,7 @@ NVF_API void inlineSelectedAt(
     bool best_effort = false,
     const std::unordered_set<IterDomain*>& uninlinable_ids = {});
 
+// Reset inlining of all tensors
+void resetInlining(Fusion* fusion);
```

Comment:
nitpick: I *imagine* this needs to be exposed for python API. But we can also wait until when that time comes before exposing the symbol.